### PR TITLE
Fixed mulitple flake8 errors

### PR DESCRIPTION
**Pull request by gonzodepedro**
Signed-off-by: Gonzalo de Pedro <gonzalo@depedro.com.ar>

This PR will fix the flake8 erros found on CI

### CODE REVIEW

**Comment by gonzodepedro** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/7790/)

File: src/rqt_reconfigure/treenode_item_model.py [hunk 1]
```diff
@@ -41,7 +41,10 @@
 
 class TreenodeItemModel(QStandardItemModel):
     """
-    This class is made only for this purpose; to hold QStandardItem instances
+
```
Comment (cottsay) [inline src/rqt_reconfigure/treenode_item_model.py]: Can you update the leading blank lines to be consistent? Looks like some of the blocks have a leading blank line at the beginning of the comment block and some don't. I don't care which way you go...

Comment (gonzodepedro) [reply src/rqt_reconfigure/treenode_item_model.py]: Done

**Comment by cottsay** [comment]
Another CI run after latest changes: [Build Status](https://ci.ros2.org/job/ci_linux/7815/)

**Pull request merged**